enterprise/server/cmd/executor/BUILD

```diff
@@ -198,6 +198,21 @@ container_layer(
     tars = ["//enterprise/server/remote_execution/containers/podman:podman-static.tar.gz"],
 )
 
+container_layer(
+    name = "podman_static_runc_layer",
+    directory = "/usr/local/bin",
```

Comment:
Just FYI I got this from the latest image

```
> docker run --rm -it --entrypoint="" gcr.io/flame-public/buildbuddy-executor-enterprise:latest which runc
/usr/bin/runc
```

So not sure if writing runc to `/usr/local/bin` will take precedence.